Godeps/_workspace/src/github.com/docker/spdystream/connection.go

```diff
@@ -328,6 +337,7 @@ func (s *Connection) Serve(newHandler StreamHandler) {
 		frameQueues[partition].Push(readFrame, priority)
 	}
 	close(s.closeChan)
+	wg.Wait()
```

Comment:
The spdystream changes will go in a separate PR to spdystream. WIP is here: https://github.com/ncdc/spdystream/pull/1